src/Microsoft.VisualStudio.Editor.Razor/ImportDocumentInfo.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Razor.Language;
+
+namespace Microsoft.VisualStudio.Editor.Razor
+{
+    public abstract class ImportDocumentInfo
+    {
+        public abstract string FilePath { get; }
+
+        public abstract RazorSourceDocument Import { get; }
```

Comment:
I added this property because for cases like in-memory default imports it didn't make sense to have an entry here with a `null` file path.

Reply:
Sorry, I don't super get this comment.